src/processLambda/app/app.py

```diff
@@ -1,68 +1,106 @@
 import boto3
-# import ffmpeg
 import os
-import json
-from botocore.client import Config
 import subprocess
 import shlex
-import re
-import time
+from boto3.dynamodb.conditions import Key
 
 SOURCE_BUCKET = os.environ['MEDIA_CAPTURE_BUCKET']
 SOURCE_PREFIX = 'captures'
+s3 = boto3.client('s3')
```

Comment:
Can we add v4 signature by default? As a good practice.
`s3 = boto3.client('s3', config=Config(signature_version='s3v4'))`